tests/integration/components/o-s-s/chip-test.ts

```diff
@@ -0,0 +1,108 @@
+      setupOnerror((err: any) => {
+        assert.equal(
+          err.message,
+          'Assertion Failed: [component][OSS::Chip] The parameter @onRemove of type function is mandatory'
+        );
+      });
+      await render(hbs`<OSS::Chip @label={{this.label}}/>`);
+    });
+
+    test('It throws an error if @label is not passed', async function (assert) {
+      setupOnerror((err: any) => {
+        assert.equal(err.message, 'Assertion Failed: [component][OSS::Chip] The @label parameter is mandatory');
+      });
+      await render(hbs`<OSS::Chip @onRemove={{this.onRemove}} />`);
+    });
+
+    test('it renders the proper label when @label is passed', async function (assert) {
+      await render(hbs`<OSS::Chip @label={{this.label}} @onRemove={{this.onRemove}} />`);
+
+      assert.dom('.upf-chip').exists({ count: 1 });
+      assert.dom('.upf-chip').hasText('Test');
+    });
+
+    test('The component is disabled if the @disabled parameter is truthy', async function (assert) {
+      await render(hbs`<OSS::Chip @label={{this.label}} @onRemove={{this.onRemove}} @disabled={{true}} />`);
+
+      assert.dom('.upf-chip').exists({ count: 1 });
+      assert.dom('.upf-chip').hasClass('upf-chip--disabled');
+    });
+  });
+  module('@maxDisplayWidth', () => {
+    test('The component adds an ellispsis if the label is wider than @maxDisplayWidth', async function (assert) {
+      await render(
+        hbs`<OSS::Chip @label="Test with a huge label sentence" @onRemove={{this.onRemove}} @maxDisplayWidth={{100}}/>`
+      );
+
+      assert.dom('.upf-chip').exists({ count: 1 });
+      assert.dom('.upf-chip span').hasStyle({
+        maxWidth: '100px',
+        overflow: 'hidden',
+        textOverflow: 'ellipsis',
+        whiteSpace: 'nowrap'
+      });
+    });
+
+    test('When the param is not set', async function (assert) {
+      await render(hbs`<OSS::Chip @label="Test with a huge label sentence" @onRemove={{this.onRemove}} />`);
+
+      assert.dom('.upf-chip').exists({ count: 1 });
+      assert.dom('.upf-chip span').doesNotHaveAttribute('style');
+      assert.dom('.upf-chip').hasText('Test with a huge label sentence');
+    });
+  });
+
+  module('On cross click event', () => {
+    hooks.beforeEach(function () {
+      this.onRemove = sinon.stub();
+    });
+    test('When clicking on the close icon, the component triggers the @onRemove method', async function (assert) {
+      await render(hbs`<OSS::Chip @label={{this.label}} @onRemove={{this.onRemove}} />`);
+      await click('.upf-chip i');
+      assert.true(this.onRemove.calledOnce);
+    });
+
+    test('When @disabled is true and clicking on the close icon, the component no triggers the @onRemove method', async function (assert) {
+      await render(hbs`<OSS::Chip @label={{this.label}} @onRemove={{this.onRemove}} @disabled={{true}} />`);
+      await click('.upf-chip i');
+      assert.false(this.onRemove.calledOnce);
```

Comment:
Not a valid test, if the function is called twice it pass the test.
You should write `assert.true(this.onRemove.notCalled);`